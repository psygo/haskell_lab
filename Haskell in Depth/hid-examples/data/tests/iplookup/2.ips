68.66.248.20
127.0.0.1
40.18.226.141
42.255.174.226
192.168.1.1