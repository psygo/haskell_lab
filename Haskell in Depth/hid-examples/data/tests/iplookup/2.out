68.66.248.20: YES
127.0.0.1: NO
40.18.226.141: YES
42.255.174.226: YES
192.168.1.1: NO
